src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -1,4 +1,5 @@
 #include "CountryInstance.hpp"
+#include <cstdint>
```

Comment:
Not needed I think.

Reply:
Although we get that include from other headers too, we do use types defined in `<cstdint>` in this file so it's not useless/wasteful. If we go for a full "include what you use" approach like Spartan advocates, then for every file we'll have every single header who's contents we need included explicitly (this doesn't result in any meaningful compile or run time slowdown, and it avoid issues when removing includes in headers that other files depend on but don't include directly themselves).